### PR TITLE
Add player move

### PR DESCRIPTION
**Pull request by TomasBorsje**
Adds Player#Move, Location#move, Tile-related utility methods in Board, Board#display

### CODE REVIEW

File: src/Game.java [hunk 1]
```diff
@@ -159,7 +159,7 @@ public static Board setupBoard() {
         for(int p = 0; p < 4; p++){
             playersList.add(new PlayerTile(CHARACTER_STARTS[p], new Player(characterCards.get(p))));
         }
-        Collections.shuffle(playersList);
+        Collections.shuffle(playersList); // Why are we shuffling here?
```
Comment (Lucas-ws) [inline src/Game.java:162]: We shuffle here so that the players are assigned a random character.

File: src/core/Board.java [hunk 2]
```diff
@@ -67,6 +67,72 @@ private void initialise(List<Estate> estates, List<List<Location>> doorways, Lis
...
+     * @return The board size
+     */
+    public int getBoardSize() {
+        return BOARD_SIZE;
+    }
+
+    /**
+     * Displays the board.
+     */
+    public void display() {
```
Comment (Lucas-ws) [inline src/core/Board.java:81]: This is fine and all but we don't always want to display the board and include a linebreak. Not sure we need a function to call a print.

File: src/Game.java (hunk 1 above)
Comment (TomasBorsje) [reply src/Game.java:162]: ok and? mind your own business

File: src/core/Location.java [hunk 3]
```diff
@@ -17,6 +19,15 @@ public int getY() {
         return this.y;
     }
 
+    public Location move(Game.Direction direction) {
+        switch(direction) {
+            case UP -> { return new Location(x,y-1); } // Up and down are inverted as printing is upside down
```
Comment (Lucas-ws) [inline src/core/Location.java:24]: Be careful here this way of writing case switch is new as of java 14 and I think Vic uses 8.

**Review by Lucas-ws — CHANGES_REQUESTED** [review]
Looks good! Just a couple of small notes. Also are you sure we need to be searching for player tiles? We know where the players are.

File: src/core/Board.java (hunk 2 above)
Comment (TomasBorsje) [reply src/core/Board.java:81]: Figured board.display() would be neater than system.out.println(board) but line break cases are fair enough. Leaving it in as a cleaner way to print with a line break, feel free to use another way of printing it if the lack of line break is necessary

File: src/core/Location.java (hunk 3 above)
Comment (TomasBorsje) [reply src/core/Location.java]: Fixed. Thanks, IntelliJ!

**Review by Lucas-ws — APPROVED** [review]

**Pull request merged**